flink-cdc-connect/flink-cdc-source-connectors/flink-connector-debezium/src/main/java/io/debezium/relational/RelationalTableFilters.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.relational;
+
+import io.debezium.config.Configuration;
+import io.debezium.relational.Selectors.TableIdToStringMapper;
+import io.debezium.relational.Selectors.TableSelectionPredicateBuilder;
+import io.debezium.relational.Tables.TableFilter;
+import io.debezium.relational.history.DatabaseHistory;
+import io.debezium.schema.DataCollectionFilters;
+
+import java.util.function.Predicate;
+
+import static io.debezium.relational.RelationalDatabaseConnectorConfig.COLUMN_BLACKLIST;
+import static io.debezium.relational.RelationalDatabaseConnectorConfig.COLUMN_EXCLUDE_LIST;
+
+public class RelationalTableFilters implements DataCollectionFilters {
```

Comment:
Add java doc and describe where is changed.

Reply:
Addressed.